lib/internal/fs/rimraf.js

```diff
@@ -231,6 +231,8 @@ function _rmdirSync(path, options, originalErr) {
   } catch (err) {
     if (err.code === 'ENOENT')
       return;
+    if (err.code === 'EACCES')
+      throw err;
```

Comment:
Should this not be the original error?

Reply:
Thanks for pointing out. Looking at how the error handling is done in the other lines, it does seem like `originalErr` should be the way to go. Is there a case where `err` will produce a different result though?

Reply:
It would produce a different stack trace.

Reply:
@aduh95 Thanks for letting me know :)